delegates.txt

```diff
@@ -60,6 +60,7 @@ Caroline Cullen (CCU)
 Chengzhong Wu (CZW)
 Chip Morningstar (CM)
 Chris Anderson (CAN)
+Chris de Almeida (CMD)
```

Comment:
Would you consider `CDA` or `CAA` to match others?

Reply:
would `CD` be ok?

Reply:
done